src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -0,0 +1,28 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.entities.Question;
+import com.team4.testingsystem.services.QuestionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path = "/question")
+public class QuestionController {
+    private final QuestionService questionService;
+
+    @Autowired
+    public QuestionController(QuestionService questionService) {
+        this.questionService = questionService;
+    }
+
+    @GetMapping("/{id}")
+    public Question getQuestion(@PathVariable("id") Long id) {
+        return questionService.getQuestionById(id);
+    }
+
+    @PostMapping("/update")
```

Comment:
/question/update URL that updates only it's status seams confusing.
Please, use another mapping.
Or it can be done in a method that makes update operation for entire question